code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -202,6 +202,20 @@
 		plane_holder.set_vis(vis,FALSE)
 		vis_enabled -= vis
 
+/mob/living/carbon/human/get_restraining_bolt()
+	var/obj/item/weapon/implant/restrainingbolt/RB
+
+	for(var/obj/item/organ/external/EX in organs)
+		RB = locate() in EX
+		if(istype(RB))
+			break
+
+	if(RB)
+		if(!RB.malfunction)
+			return TRUE
```

Comment:
If a human has two bolts implanted, only one broken, and this happens to find the broken one first, the working one won't function at all until the broken one is removed.

Reply:
Fixt.